crates/cheatnet/src/state.rs

```diff
@@ -202,12 +210,18 @@ impl StateReader for DictStateReader {
     }
 }
 
+pub enum WarpedContract {
+    Warped(Felt252),
+    Unwarped,
+}
```

Comment:
Can we come up with a different name? We'll be reusing this probably

Reply:
In that case would it have to be generic? As in, 
```rust
pub enum CheatedContract<T> {
   Cheated(T), 
   Uncheated,
}
```

Because some of the cheats (`prank`, `spoof`) require types other than Felt.

Reply:
Alternative name: `CheatInfo`. Wdyt?